### ISSUE TITLE
JavaScript heap out of memory

### ISSUE THREAD

**Issue by jakalkhalili** [Microsoft/BosqueLanguage#106]
**Describe the bug**
When I try to convert the type of Any type to Int, it causes a long time of reading app code and after that I get JS stacktrace.

**To Reproduce**
Here is the code that triggers this error
```
function convert(x: Any) : Int {
        return x->as[Int]();
}

entrypoint function main() : Int {
    return convert(3);
}

```

**Expected behavior**
It should convert Any type to Int.

**Screenshots**
https://imgur.com/a/rgeL8DO

**Desktop (please complete the following information):**
 - OS: Arch Linux x64 5.0.7-arch1-1-ARCH
 - Bosque Version: master branch

**Additional context**
Nothing

**Comment by mrkmarron** [comment]
Hi @jakalkhalili, thanks for the bug report. I can reproduce it and should have a fix shortly.

**Comment by mrkmarron** [comment]
This should be fixed in PR #120. Closing for now, feel free to reopen if you are still seeing problems.
